api/prisma/migrations/20240904131219_remove_dummy_event_type/migration.sql

```diff
@@ -0,0 +1,14 @@
+/*
+  Warnings:
+
+  - The values [DUMMY] on the enum `EventType` will be removed. If these variants are still used in the database, this will fail.
+
+*/
+-- AlterEnum
+BEGIN;
+CREATE TYPE "EventType_new" AS ENUM ('REQUEST_CONTROL');
+ALTER TABLE "Event" ALTER COLUMN "type" TYPE "EventType_new" USING ("type"::text::"EventType_new");
+ALTER TYPE "EventType" RENAME TO "EventType_old";
+ALTER TYPE "EventType_new" RENAME TO "EventType";
```

Comment:
After you've renamed EventType_new to EventType do you need to update the column to use this? Or does postgres track the column renaming?

Reply:
Ah good so it tracks the name change